controllers/hbase_controller.go

```diff
@@ -578,8 +578,14 @@ func (r *HBaseReconciler) ensureStatefulSet(hb *hbasev1.HBase,
 		}
 		return nil, false, err
 	}
+	expected, expectedRevision := r.statefulSet(hb, stsName, cmName, ss, *actual.Spec.Replicas)
+
+	r.Log.Info("updating revision", "?", expectedRevision != actual.Annotations[HBaseControllerRevisionKey])
```

Comment:
what is `?` as second arg to Info?

Reply:
Honestly, just a bogus filler. This reconciler log requires 3 arguments, and in some of the code we use "name" as a string arg for the 2nd. I can revert to using "name", it just made reading the logs easier to have "?" because it reads like:

2021-05-03T09:42:12.067-0700 INFO controllers.HBase updating revision {"?": false}

Reply:
can't you call it something that it actually is? like in this case `sameRevision: false`

Reply:
ACK